### ISSUE TITLE
When will a new version be released?

### ISSUE THREAD

**Issue by phdesign** [alexis-mignon/python-flickr-api#79]
Hi, I just spent a few hours tracking down a bug with Windows not reading the entire file to find it's been fixed already. Are you considering a new release soon?

Kind regards,

Paul.

**Comment by tomquirk** [comment]
@alexis-mignon just found this project, really fantastic work. 

Can you let us know if you're planning on maintaining it?

**Comment by Flimm** [comment]
Thank you @tomquirk and @alexis-mignon  for your recent commits to this code-base. I'm excited about Python 3 support.

It looks like @alexis-mignon is the only one who is able to upload a new version of [`flickr_api` to Pypi](https://pypi.python.org/pypi/flickr_api).

**Comment by tomquirk** [comment]
Indeed. IMO, it would be nice to get a basic test suite in place before another release. If I have time I'll try to put something together within the next 2 weeks. At the very least, play with it enough to be happy with the project's stability 😂

**Comment by Flimm** [comment]
I am eagerly awaiting a Python 3 compatible release of `flickr_api` on PyPI, so that I can upgrade my project to Python 3 too.

**Comment by tomquirk** [comment]
@alexis-mignon, lets get it done! Tests can wait 😄

**Comment by alexis-mignon** [comment]
If you think it's ok. Can you do it?

**Comment by Flimm** [comment]
Woot! Now I can upgrade [my project](https://github.com/Flimm/backup-all-my-flickr-photos) to Python 3 too.

https://pypi.python.org/pypi/flickr_api
